Source/RuntimeImageLoader/Private/RuntimeImageLoader.cpp

```diff
@@ -290,6 +295,59 @@ void URuntimeImageLoader::LoadImagePixels(const FInputImageDescription& InputIma
     Requests.Enqueue(Request);
 }
 
+void URuntimeImageLoader::LoadGIF(const FString& GIFFilename, UAnimatedTexture2D*& OutTexture, bool& bSuccess, FString& OutError, FLatentActionInfo LatentInfo, UObject* WorldContextObject)
+{
+    UAsyncGIFLoader* AnimatedGifLoader = NewObject<UAsyncGIFLoader>();
+    check (IsValid(AnimatedGifLoader));
```

Comment:
we need to unify API logic. As you added `URuntimeGifReader` please use it here as well